pkg/services/ngalert/image/service.go

```diff
@@ -114,16 +103,16 @@ func NewScreenshotImageServiceFromCfg(cfg *setting.Cfg, db *store.DBstore, ds da
 //
 // The alert rule must be associated with a dashboard panel for a screenshot to be
 // taken. If the alert rule does not have a Dashboard UID in its annotations,
-// or the dashboard does not exist, an ErrNoDashboard error is returned. If the
+// or the dashboard does not exist, an models.ErrNoDashboard error is returned. If the
```

Comment:
The Go doc-style feels like it's conflicting with English here, I'm not sure which is grammatically correct :thinking: 

- `an models.ErrNoDashboard` -> "models" is just a qualifier and is not pronounced -> stay with `an`, "an ErrNoDashboard"
- `a models.ErrNoDashboard` -> "models" is part of the name just as the actual type name and is pronounced -> use `a` instead of `an`